python/ray/_private/services.py

```diff
@@ -265,24 +264,6 @@ def address(ip_address, port):
     return ip_address + ":" + str(port)
 
 
-def new_port(lower_bound=10000, upper_bound=65535, denylist=None):
```

Comment:
the `denylist` logic here was wholly unused... preferring to just use `randint` at the few callsites